src/DataPersister/DataPersisterInterface.php

```diff
@@ -33,6 +33,8 @@ public function supports($data): bool;
      * Persists the data.
      *
      * @param mixed $data
+     *
+     * @return object|void Void will not be supported in API Platform 3, an object should always be returned
```

Comment:
I think it might be better if we add deprecation errors where this method is called in API Platform?

Reply:
Doing it in `ChainDataPersister` should be enough. If we deprecate returning `void`, please remove it from the supported types and only keep it in the comment.

Reply:
Ok you already made the changes, so keep them :)

Reply:
> Doing it in ChainDataPersister should be enough.

It's not enough because `ChainDataPersister` is only an implementation which might or might not be used by the user.